examples/jar-from-manifest/pom.xml

```diff
@@ -38,15 +38,14 @@
     <packaging>jar</packaging>
 
     <properties>
-        <mainClass>com.cloudogu.versionname.App</mainClass>
+        <mainClass>com.cloudogu.versionname.AppWithoutDeps</mainClass>
```

Comment:
If i run this example i get a main class not found error. Reverting the change for this line fixes the problem for me.

Reply:
Nice finding! The integration test on Jenkins would also have found it, if it had been run.
We now have Jenkins building also PRs and it's green after I fixed the change requested.
Thanks for the review.